### DISCUSSION TITLE
How to use RadixUI theme in Shadcn

### DISCUSSION THREAD

**Discussion by piszczu4** [shadcn-ui/ui#1723] [Q&A]
Is it possible to somehow easy integrate radix UI theme in shadcn or I need to manually copy radix css, convert it to tailwind and apply to my shadcn components?

**Comment by nagy-nabil** [comment]
from this comment [https://github.com/radix-ui/themes/discussions/95#discussioncomment-7227810](https://github.com/radix-ui/themes/discussions/95#discussioncomment-7227810) i don't think they are meant to be used together and would need to do it manually
